packages/flutter/lib/src/widgets/color_filter.dart

```diff
@@ -11,7 +11,52 @@ import 'framework.dart';
 
 /// Applies a [ColorFilter] to its child.
 ///
+/// Use the [ColorFilter.mode] constructor to apply a [Color] using a [BlendMode].
```

Comment:
Maybe remove this line? It's a bit weird to have this line immediately after the summary.